app/components/media/title_component.html.erb

```diff
@@ -0,0 +1,3 @@
+    <h3 class="pt-10 sm:pt-0 text-lg font-semibold <%= classes %>">
+      <%= @media[:title] %>
+    </h3>
```

Comment:
Linter: Final newline

Reply:
Addressed this in https://github.com/TheOdinProject/theodinproject/pull/3926/commits/db8e1fe866fa7c008677db7ef08b1ca782842597